   ml_ops/sm-datazone_import/import-sagemaker-domain.py

```diff
@@ -398,9 +418,27 @@ def _associate_fed_role(self):
             else:
                 print(f"Caught error: {repr(e)}")
 
+    def _cross_account_action(self):
+        # Decision to link domain from X-account. Required to assumeRole from child-account containing SM domain.
+
+        decision = input("Need to import SM profiles from a X-account? [y/n]: ")
+        if decision == "y":
+            x_account_role = input(
+                "X-account role ARN to assume? This would be the account that originally created"
```

Comment:
"X-account" -> terminology that customers would understand better. i.e. 
"Need to import SageMaker profiles from a different account?"
"Secondary account role ARN to assume"

what account is this asking for exactly? the DataZone Domain account or the SageMaker Domain account? i.e. above line mentions "SM profiles" but below mentions "DataZone domain"

Reply:
also avoid abbreviating SageMaker as SM to avoid potential user confusion

Reply:
Ack. Will clean this up.